datafusion/sqllogictest/test_files/window.slt

```diff
@@ -4892,7 +4892,7 @@ DROP TABLE t1;
 statement ok
 CREATE TABLE t1(v1 BIGINT);
 
-query error DataFusion error: Execution error: Expected a signed integer literal for the second argument of nth_value, got v1@0
+query error DataFusion error: This feature is not implemented: There is only support Literal types for field at idx: 1 in Window Function
```

Comment:
The original error message is more specific. You can transform it by using `map_err` when invalid input is provided.